Aptoide/aptoide-client-v8
# [AN-1646] Add social info bar to the aggregated minimal cards.

What does this PR do?

   This pull-request adds the social info bar (likes preview + number of comments + last comment made) to the Aggregated card shares (sub-cards - minimal cards).

Where should the reviewer start?

- [x] Aggregated Social install card
- [x] Aggregated Social article card
- [ ] Aggregated Social video card
- [x] Aggregated Social store latest card

How should this be manually tested?

  Open Aptoide v8 > Apps Timeline > Find aggregated cards > check if it has likes & comments and if they show up in the social info bar. If not, make a comment/like in it.

What are the relevant tickets?

Tickets related to this pull-request: [AN-1646](https://aptoide.atlassian.net/browse/AN-1646).

Screenshots (if appropriate)

N/A
Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
